root/static/scripts/tests/Control/URLCleanup.js

```diff
@@ -381,6 +381,10 @@ const testData = [
                      input_url: 'https://music.apple.com/jp/album/uchiagehanabi-single/1263790414',
             expected_clean_url: 'https://music.apple.com/jp/album/1263790414',
   },
+  {
+                     input_url: 'https://beta.music.apple.com/ca/artist/imposs/205021452',
+            expected_clean_url: 'https://music.apple.com/ca/artist/205021452',
+  },
```

Comment:
Tests are usually grouped by entity type, this test should probably follow the first one for Apple Music.